src/main/java/seedu/address/logic/commands/AddTaskCommand.java

```diff
@@ -24,19 +24,19 @@
 public class AddTaskCommand extends Command {
 
     /* Message printed if wrong usage */
-    public static final String MESSAGE_USAGE = "addt" + ": Adds a task to the NUS Classes. "
+    public static final String MESSAGE_USAGE = "addt" + ": Adds a task to NUS Classes. "
             + "Parameters: "
             + PREFIX_TASKNAME + "TASKNAME "
             + PREFIX_DATETIME + "DATETIME [, ENDDATETIME]"
-            + PREFIX_TAG + "TAG "
-            + PREFIX_LINK + "LINK "
-            + PREFIX_RECURRING + "PERIOD RECURRENCE\n"
+            + " [" + PREFIX_TAG + "TAG] "
+            + " [" + PREFIX_LINK + "LINK] "
+            + " [" + PREFIX_RECURRING + "PERIOD RECURRENCE]\n"
             + "Example: " + "addt" + " "
             + PREFIX_TASKNAME + "John Doe "
             + PREFIX_DATETIME + "25-12-2022 1800 "
-            + PREFIX_TAG + "CS2103T"
-            + " [" + PREFIX_LINK + "https://...]"
-            + " [" + PREFIX_RECURRING + "5 5]\n"
+            + PREFIX_TAG + "CS2103T "
+            + PREFIX_LINK + "https://[...] "
```

Comment:
Since this is an example we shouldn't put the [ ] brackets? Just leave it as https://... else is it possible they get confused thinking its optional

Reply:
👌